CMakeLists.txt

```diff
@@ -706,11 +719,15 @@ if(EXECUTORCH_BUILD_VULKAN)
 endif()
 
 # Top-level interface targets.
-add_library(executorch_backends INTERFACE)
 
 # A target containing all configured backends.
+add_library(executorch_backends INTERFACE)
 target_link_libraries(executorch_backends INTERFACE ${_executorch_backends})
 
+# A target containing all configured extensions.
+add_library(executorch_extensions INTERFACE)
```

Comment:
can you please include these in the install() call alongside executorch and executorch_core so that we don't have to coordinate with the EXPORT PR?

Reply:
I've updated to install these targets. Because of the order of declarations in the file, I can't directly add to the existing executorch / executorch_core install, as it is above many of the backend declarations. To avoid causing a bunch of merge conflicts by rearranging it, I've added an install for these new targets alone. I can add the export directive once #8954 is merged.

Reply:
I've updated executor_runner to use the top-level backends and extensions targets. I'll update examples with the export changes, as well.